apps/minifront/src/components/swap/asset-out-box.tsx

```diff
@@ -56,33 +56,55 @@ export const AssetOutBox = ({ balances }: AssetOutBoxProps) => {
       <div className='mb-2 flex items-center justify-between gap-1 md:gap-2'>
         <p className='text-sm font-bold md:text-base'>Swap into</p>
       </div>
-      <div className='flex items-center justify-between gap-4'>
-        {simulateOutResult ? (
-          <Result result={simulateOutResult} />
-        ) : (
-          <EstimateButton simulateFn={simulateSwap} loading={simulateOutLoading} />
-        )}
-        <AssetOutSelector
-          balances={balances}
-          assetOut={matchingBalance}
-          setAssetOut={setAssetOut}
-        />
-      </div>
-      <div className='mt-[6px] flex items-start justify-between'>
-        <div />
-        <div className='flex items-start gap-1'>
-          <img src='./wallet.svg' alt='Wallet' className='size-5' />
-          <ValueViewComponent view={matchingBalance} showIcon={false} />
+      <div className='flex justify-between gap-4'>
+        <div className='flex items-start justify-start'>
+          {simulateOutResult ? (
+            <Result result={simulateOutResult} />
+          ) : (
+            <EstimateButton simulateFn={simulateSwap} loading={simulateOutLoading} />
+          )}
+        </div>
+        <div className='flex flex-col'>
+          <div className='ml-auto w-auto shrink-0'>
+            <AssetOutSelector assetOut={matchingBalance} setAssetOut={setAssetOut} />
+          </div>
+          <div className='mt-[6px] flex items-start justify-between'>
+            <div />
+            <div className='flex items-start gap-1'>
+              <img src='./wallet.svg' alt='Wallet' className='size-5' />
+              <ValueViewComponent view={matchingBalance} showIcon={false} />
+            </div>
+          </div>
```

Comment:
The CSS in this section was kinda wonky. This revision separates the content on the left vs right better.